pkgs/applications/misc/decker/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, SDL2
+, SDL2_image
+, unixtools
+, multimarkdown
+}:
+
+stdenv.mkDerivation rec {
+  pname = "decker";
+  version = "1.12";
+
+  src = fetchFromGitHub {
+    owner = "JohnEarnest";
+    repo = "Decker";
+    rev = "v${version}";
+    sha256 = "sha256-NZudcJIxLq9dk7ayT8RdpOh6ObP0FANZu9CI0UCyKjg=";
+  };
+
+  buildInputs = [
+    SDL2
+    SDL2_image
+    multimarkdown
+    unixtools.xxd
+  ];
+
+  doCheck = true;
+
```

Comment:
```suggestion
, fetchFromGitHub
, fetchpatch
, SDL2
, SDL2_image
, unixtools
, multimarkdown
}:

stdenv.mkDerivation rec {
  pname = "decker";
  version = "1.12";

  src = fetchFromGitHub {
    owner = "JohnEarnest";
    repo = "Decker";
    rev = "v${version}";
    sha256 = "sha256-NZudcJIxLq9dk7ayT8RdpOh6ObP0FANZu9CI0UCyKjg=";
  };

  buildInputs = [
    SDL2
    SDL2_image
    multimarkdown
    unixtools.xxd
  ];

  doCheck = true;

  patches = [
    (fetchpatch {
      url = "https://patch-diff.githubusercontent.com/raw/JohnEarnest/Decker/pull/53.diff";
      sha256 = "sha256-7X0KBuNV+yp0HBTJ17WfJhFudv6QqFn+nYbADV1A+BA=";
    })
  ];

```

Reply:
Added the patch. I can't confirm the fix though since I don't have a Mac.

Reply:
Neither do I lol, but the ofborg builds look happy now :tada: